softdevice_controller/CHANGELOG.rst

```diff
@@ -107,7 +110,7 @@ Changes
 Bugfixes
 ========
 
-* Fixed an issue in master role which could cause disconnects if there were scheduling conflicts while doing a control procedures with an instant (DRGN-11222).
+Fixed an issue in master role which could cause disconnects if there were scheduling conflicts while doing a control procedures with an instant (DRGN-11222).
```

Comment:
What is "master role" and "an instant". Also, typo: `doing [a] control procedures`

Reply:
We should not change old release notes (that's a general comment for the whole PR).